scripts/seastar-addr2line

```diff
@@ -80,7 +80,7 @@ class BacktraceResolver(object):
             path = "\S+"
             token = f"(?:{path}\+)?{addr}"
             full_addr_match = f"(?:({path})\+)?({addr})"
-            self.oneline_re = re.compile(f"^((?:.*(?:(?:at|backtrace):?|:))?(?:\s+))?({token}(?:\s+{token})*)\s*$", flags=re.IGNORECASE)
+            self.oneline_re = re.compile(f"^((?:.*(?:(?:at|backtrace):?|:))?(?:\s+))?({token}(?:\s+{token})*)[\s\)]*$", flags=re.IGNORECASE)
```

Comment:
If there is a nested exception, there could be more text in the line. Maybe we should allow arbitrary characters at the end.

Even better would be to capture all the backtraces separated by arbitrary text, but that can be solved later.

Reply:
It's possible, but I'd still require the "backtrace" or "at" keyword since there are scattered single addresses printed, e.g. by ASAN that we should not detect as backtraces IMO.